### DISCUSSION TITLE
What's next?

### DISCUSSION THREAD

**Discussion by arthur-fontaine** [AntoineW/luge#13] [General]
Is it possible to add a "What's next" section in the README? I like the idea of this project, and I'd like to contribute to this project, but I don't know what to do.

**Comment by AntoineW** [comment]
Thanks Arthur! Do you have example of such section on other projects? I can't find one that suits me well.
Anyway, right now what I'm the most looking for is bug reports, overall feedback and new feature suggestion. I do have a personnal roadmap, but the current version is already working well for what it was created for.

**Reply by arthur-fontaine** [reply]
I don't have an example, I saw that a day, but I don't remember in which repository. But I think you can simply write something like that at the bottom of the README:
```md
## What's next?
- [ ] Something to do
- [ ] Another thing to do
- [x] Something already did
```

Do you think that adding Definition Types for Typescript could be a good "feature"?

**Reply by AntoineW** [reply]
I might be wrong but I would not consider "adding Definition Types" as a "feature" as I assume it won't add any value for the end user. 

The next things I might want to add to luge would rather be a better support for JS frameworks (React, Vue, ...) and a more robust API to be able to use luge without using the html data- attributes.

**Reply by arthur-fontaine** [reply]
Ok, I will try to work on that when I could
